packages/fxa-shared/metrics/navigation-timing.ts

```diff
@@ -15,18 +15,6 @@ const NAV_ENTRY_TYPE = 'navigation';
 
 // export for testing
 export const sendFn = () => {
-  if ('keepalive' in new Request('')) {
```

Comment:
esbuild got hungup on where `Request` was defined. Since firefox doesn't support `keepalive` I think we can just remove this block.